client_plugin/drivers/vfile/kvstore/etcdops/etcdops.go

```diff
@@ -893,6 +916,7 @@ func (e *EtcdKVS) createEtcdClient() *etcdClient.Client {
 func addrToEtcdClient(addr string) (*etcdClient.Client, error) {
 	// input address are RemoteManagers from docker info or ManagerStatus.Addr from docker inspect
 	// in the format of [host]:[docker manager port]
+	etcdClientPort, _ := getEtcdPorts()
```

Comment:
This addrToEtcdClient function is called inside createEtcdClient function and will be called for almost every ETCD operations. So calling getEtcdPorts inside this function is not efficient (means everytime we read ENVs again although they are installation flags).
I suggest that we should call getEtcdPorts at the very beginning and then store the information inside the local EtcdKVS struct. And later in addrToEtcdClient we just need to grab this information locally, instead of reading the ENV flags again and again.

Reply:
@luomiao I have addressed your comments, please take a look. Thanks!